frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -317,15 +317,15 @@ export function SurveyReleaseSummary({
                     <span className="simple-tag tag-light-blue text-primary-alt">{survey.conditions.selector}</span>
                 </div>
             )}
-            {(targetingFlagFilters?.groups?.[0].properties?.length || 0) > 0 && (
+            {(targetingFlagFilters?.groups?.[0]?.properties?.length || 0) > 0 && (
                 <div className="flex flex-row font-medium gap-1">
                     <span>User conditions:</span>{' '}
                 </div>
             )}
             {targetingFlagFilters?.groups?.map((group, index) => (
                 <>
                     {index > 0 && <div className="text-primary-alt font-semibold text-xs ml-2 py-1">OR</div>}
-                    {group.properties.map((property, idx) => (
+                    {group.properties?.map((property, idx) => (
```

Comment:
When group is present but properties are undefined, this leads to a wonky state:

<img width="1255" alt="image" src="https://github.com/PostHog/posthog/assets/7115141/50e4fdcc-7cf4-4128-a287-e400b4fe3cd1">

I.e release summary says there's some criteria, but there's no criteria.